package.json

```diff
@@ -6,7 +6,8 @@
   "license": "MIT",
   "main": "dist/cjs.js",
   "files": [
-    "dist"
+    "dist",
+    "options.json"
```

Comment:
I'm 💯 sry about this, your initial way of handling this file via `--copy-files` in the `build` script is the better solution, please revert :) 👍